Tests/InkTests/LinkTests.swift

```diff
@@ -47,6 +47,11 @@ final class LinkTests: XCTestCase {
         let html = MarkdownParser().html(from: "[He_llo](/he_llo)")
         XCTAssertEqual(html, "<p><a href=\"/he_llo\">He_llo</a></p>")
     }
+
+    func testFakeLink() {
```

Comment:
Could this be named a bit more descriptively? For example `testPartialLink` or `testUnterminatedLink`. Just so that it'll be easier to see what failed if a regression is ever caused. Also, like you suggested, please add this test on Linux as well (it can be done as a separate commit on this PR, no worries). Thanks a lot for this @john-mueller! 👍

Reply:
Renamed and added Linux test. Took two commits, because stringly-typed... 🤦‍♂️